project/src/main/kotlin/org/ionproject/core/klass/handlers.kt

```diff
@@ -0,0 +1,61 @@
+package org.ionproject.core.klass
```

Comment:
Update API according to the return of this code. https://github.com/i-on-project/core/blob/docs/gh-6-read-api-spec/docs/api/classes.md (properties of course are not present in the returned object) (Class Collection)

Reply:
Also, the links section returns "page=1&size=5" when the name presented in the "search" action is page/limit, I would expect to see limit instead of size.

Reply:
Also, program throws exception if I use incorrect query params like "classes?page=1&size=-1" and when doing a request like this "classes?page=100000&size=10000" it should return 404 instead of :  {
      "rel": "self",
      "href": "/v0/courses/1/classes?page=100000&size=10000"
    },